icons/reactquery.svg

```diff
@@ -0,0 +1 @@
.641 1.317-1.248.517-1 .651-2.315.459-3.94a18.514 18.514 0 0 0-.422-2.25.992.992 0 0 0 .234-.056c1.83-.677 3.126-1.404 3.91-2.236.467-.495.773-1.046.883-1.656.116-.644.002-1.29-.31-1.897-.517-1-1.465-1.788-2.817-2.412a14.089 14.089 0 0 0-1.906-.7c.026-.068.046-.142.06-.218.391-2.115.474-3.738.221-4.916-.15-.7-.424-1.271-.842-1.684C17.923.213 17.364 0 16.738 0c-1.033 0-2.119.526-3.279 1.525-.473.408-.964.898-1.475 1.473a1.208 1.208 0 0 0-.136-.164C10.405 1.399 9.189.508 8.15.164A2.964 2.964 0 0 0 7.146 0zm.141 1.129c1.479 0 3.36 1.565 5.647 4.697.05.07.078.157.078.246-.002.212-.157.383-.346.381l-.12-.002a30.248 30.248 0 0 0-2.808.113 31.98 31.98 0 0 0-3.363.489c-.183.036-.359-.1-.395-.305l-.02-.111c-.627-3.672-.185-5.508 1.327-5.508zm9.514 0c.535 0 .947.28 1.232.838.745 1.457.476 4.105-.808 7.945a.387.387 0 0 1-.153.201c-.165.106-.377.04-.47-.146l-.059-.12a39.023 39.023 0 0 0-1.502-2.708 40.442 40.442 0 0 0-2.074-3.067.423.423 0 0 1 .037-.543l.076-.076c1.588-1.55 2.828-2.325 3.72-2.324zm-6.645 6.494h3.69c.226 0 .436.136.549.356l1.847 3.6a.778.778 0 0 1 0 .702l-1.847 3.6a.624.624 0 0 1-.55.353h-3.689a.62.62 0 0 1-.547-.353l-1.85-3.6a.77.77 0 0 1 0-.703l1.85-3.6a.622.622 0 0 1 .547-.355zm-2.295.283a.33.33 0 0 1 .176.053.411.411 0 0 1 .125.533L8.1 8.61a39.331 39.331 0 0 0-1.366 2.823 42.764 42.764 0 0 0-1.353 3.558c-.066.2-.265.305-.447.237l-.1-.04C1.57 13.945.33 12.58 1.114 11.09c.766-1.456 2.993-2.516 6.685-3.18a.315.315 0 0 1 .062-.004zm2.752.604a.62.62 0 0 0-.547.353l-1.392 2.715a.777.777 0 0 0 0 .701l1.392 2.715c.113.22.32.356.547.356h2.776a.625.625 0 0 0 .549-.356l1.39-2.715a.777.777 0 0 0 0-.7l-1.39-2.716a.623.623 0 0 0-.55-.353h-2.775zm8.319.244a.326.326 0 0 1 .138.02l.098.037c3.264 1.242 4.506 2.608 3.723 4.097-.767 1.456-2.996 2.518-6.688 3.182a.33.33 0 0 1-.238-.049.411.411 0 0 1-.125-.533l.062-.117c.504-.945.96-1.887 1.368-2.823.48-1.105.93-2.293 1.351-3.56a.36.36 0 0 1 .31-.254zm-7.825.725h1.79c.226 0 .434.134.546.353l.897 1.746a.778.778 0 0 1 0 .703l-.897 1.744a.621.621 0 0 1-.547.354h-1.789a.621.621 0 0 1-.546-.354l-.899-1.744a.778.778 0 0 1 0-.703l.899-1.746a.621.621 0 0 1 .546-.353zm.475.927a.622.622 0 0 0-.547.354l-.42.816a.78.78 0 0 0 0 .705l.42.817a.624.624 0 0 0 .547.355h.838a.627.627 0 0 0 .549-.355l.42-.817a.78.78 0 0 0 0-.705l-.42-.816a.625.625 0 0 0-.55-.354h-.837zm.42.914a.54.54 0 0 1 .473.305.671.671 0 0 1 0 .61.538.538 0 0 1-.473.304.539.539 0 0 1-.473-.305.671.671 0 0 1 0-.609.538.538 0 0 1 .473-.305zm-4.926 2.516a.32.32 0 0 1 .051 0c.11.01.215.08.273.197l.06.12c.481.95.982 1.853 1.503 2.708a40.43 40.43 0 0 0 2.074 3.067.425.425 0 0 1-.039.543l-.076.076c-2.54 2.48-4.192 2.976-4.953 1.486-.745-1.457-.476-4.107.808-7.947a.39.39 0 0 1 .155-.201.305.305 0 0 1 .144-.05zM17.822 16.8c.157.004.295.129.326.308l.02.112c.628 3.672.187 5.51-1.324 5.51-1.479 0-3.363-1.568-5.649-4.7a.416.416 0 0 1-.078-.246c.002-.212.157-.38.346-.379h.119c.963.006 1.9-.03 2.809-.111a31.944 31.944 0 0 0 3.361-.488h.002a.324.324 0 0 1 .068-.006z"/></svg>
```

Comment:
It looks like you changed the dimensions of the SVG to 24x24 pixels. As the logo isn't square it should only be 24 pixels in one dimensions (the width in this case). Can you update the SVG accordingly?

Reply:
Sure... I will start working on it....